modules/virt-vmware-comparison.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * virt/about_virt/about-virt.adoc
+
+:_mod-docs-content-type: CONCEPT
+
+[id="virt-vmware-comparison_{context}"]
+= Comparison of {VirtProductName} with VMware vSphere
+
+{VirtProductName} is conceptually different from VMware vSphere, much of its functionality comes from the underlying {product-title}, and it therefore does not offer direct alternatives to individual vSphere concepts and components. That said, if you are familiar with VMware vSphere, the following table lists the most common vSphere concepts and outlines what {VirtProductName} components you can use to accomplish a similar task.
+
+.Mapping of vSphere concepts to their closest {VirtProductName} counterparts
+[options="header"]
+[cols="2,2,3"]
+|===
+|vSphere concept |{VirtProductName} |Explanation
+
+|Datastore
+|Persistent volume (PV){nbsp}+ +
+Persistent volume claim (PVC)
+|Stores VM disks. A persistent volume represents existing storage and is attached to a VM through a persistent volume claim. When created with the `ReadWriteMany` (RWX) access mode, persistent volume claims can be mounted by multiple VMs simultaneously.
+
+|Dynamic Resource Scheduling (DRS)
+|Pod eviction policy{nbsp}+ +
+Descheduler
+|Provides active resource balancing. A combination of pod eviction policies and a descheduler allows VMs to be live migrated to more appropriate nodes to keep node resource utilization manageable.
+
+|NSX
+|Multus{nbsp}+ +
+OVN-Kubernates{nbsp}+ +
+Third-party container network interface (CNI) plug-ins
+|Provides overlay network configuration. There is no direct equivalent for NSX in {VirtProductName}, depending on the desired functionality, users can either rely on the OVN-Kubernates network provider, or install certified third-party CNI plug-ins.
```

Comment:
A few nits and questions:

* The [IBM Style Guide](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#d) says no to "desired":
> **desired** _adjective_
> Do not use. Use _required_ or _that you want_.
* Should there be an article (like "an") in "Provides overlay network configuration"?
* Typo in OVN-Kubernetes.
* The second sentence should be broken up into multiple sentences or shortened.

Suggestion to take or leave:
```suggestion
|Provides overlay network configuration. There is no direct equivalent for NSX in {VirtProductName}, but you can use the OVN-Kubernetes network provider or install certified third-party CNI plug-ins.
```